mlx/traceability/assets/traceability.js

```diff
@@ -71,11 +71,12 @@ $(document).ready(function () {
 
 // item
 jQuery.fn.extend({
-    addCollapseButton: function (admonition) {
+    addCollapseButton: function (admonition, anchorId) {
         var relations = $(this);
 
         if (relations.children().length > 0) {
-            if (admonition.parent().hasClass('collapse')) {
+            const itemDiv = admonition.parent()
+            if (itemDiv.hasClass('collapse') && itemDiv.attr('id') != anchorId) {
```

Comment:
Extra brackets to improve readability and to be sure of operator precedence.

Reply:
Oops. Added in 0a6435b9a14889ef77a8908852b95241100e06de